src/home_robot/home_robot/utils/data_tools/writer.py

```diff
@@ -91,7 +91,7 @@ def fix_data(self, data):
                 new_data[k] = v
         return new_data
 
-    def flatten_dict(self, key, data):
+    def flatten_dict(self, key, data) -> Dict:
```

Comment:
hard to do types here right? maybe we dont need hints. but key = str

Reply:
Yeah the values range from str, Dict, Lists of ndarrays. I will add key types only since that is consistent.